libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginUsernamePasswordFragment.java

```diff
@@ -147,6 +147,7 @@ protected void setupContent(ViewGroup rootView) {
 
         mUsernameInput = rootView.findViewById(R.id.login_username_row);
         mUsernameInput.setText(mInputUsername);
+        mUsernameInput.setContentDescription(getString(R.string.username));
```

Comment:
Please don't use `setContentDescription` for this – per the docs, this will announce the value via the screenreader, which we don't want. 

I'm working on a solution for this right now using `getTag()`/`setTag()` that should work for our purposes – I'll tag you in the PR when the method is added to the helper library?

Reply:
Thanks for pointing this out, I totally forgot this!

Reply:
> I'll tag you in the PR when the method is added to the helper library? 

@jkmassel Yeah please do! Think that'll be ready today? I'll have trouble finding the element otherwise

Reply:
It was sort of a bust (I was trying to use it in a `RecyclerView` and that....doesn't seem to work).

Can you use `onView(withId(R.id.login_username_row))` to find the row? Then write a matcher to fetch its value?

Reply:
> Can you use onView(withId(R.id.login_username_row)) to find the row? Then write a matcher to fetch its value?

Hmm think I initially had this and was having trouble setting text on this yesterday, let me give it another go

Reply:
@jkmassel still had trouble with `onView(withId(R.id.login_username_row))`, I used the hint instead of content description for `WPLoginInputRow`. Think this works a bit better?

Reply:
I'd be very nervous to adjust the `hint` property – it's user-visible, and it ought to be localized. Neither of those attributes make it good candidates for this sort of lookup, unfortunately 😞 

My suggestion above was somewhat incomplete – `onView(withId(R.id.login_username_row))` will get you the row, but you still need to find the `TextInputEditText` inside of it. _That_ should allow you to properly populate the text field, as I suspect that `WPLoginInputRow` doesn't implement the even forwarding required to make that work.

Let me know if that helps?